diff --git a/lib/toaster/powershell.js b/lib/toaster/powershell.js
index c140c63f72eb10dc4296c46e9b57e374eafea653..529256830adab06995185f4793e32e0db6a4a8b9 100644
--- a/lib/toaster/powershell.js
+++ b/lib/toaster/powershell.js
@@ -44,7 +44,7 @@ function generate(xmlString, keepalive){
                 `$toast.Tag = "${this.options.uniqueID}"` + EOL + `$toast.Group = "${this.options.uniqueID}"` + EOL;
                    
     if (this.options.expiration)
-      template += `$toast.expiration = "${new Date(this.options.expiration).toISOString()}"` + EOL;
+      template += `$toast.ExpirationTime = "${new Date(this.options.expiration).toISOString()}"` + EOL;
 
     if (this.eventNames().length > 0) { 
       //Register events
